inst/extdata/mass-balance-settings.json

```diff
@@ -0,0 +1,100 @@
+{
+    "MassBalancePlots": [
+        {
+            "Name": "Name of mass balance sub-section (optional)",
+            "Molecules": [
+                "Aciclovir"
+            ],
+            "Groupings": [
+                {
+                    "Name": "Intracellular",
+                    "Include": [
+                        "Organism|**|Intracellular|Aciclovir*"
+                    ],
+                    "Exclude":[
+                        "Organism|Lumen|*|Aciclovir*"
+                    ]
+                },
+                {
+                    "Name": "Plasma & Blood Cells",
+                    "Include": [
+                        "Organism|**|Plasma|Aciclovir*",
+                        "Organism|**|BloodCells|Aciclovir*"
+                    ],
+                    "Exclude":[
+                        "Organism|Lumen|*|Aciclovir*"
+                    ]
+                },
+                {
+                    "Name": "Interstitial",
+                    "Include": [
+                        "Organism|**|Interstitial|Aciclovir*"
+                    ],
+                    "Exclude":[
+                        "Organism|Lumen|*|Aciclovir*"
+                    ]
+                },
+                {
+                    "Name": "Lumen",
+                    "Include": [
+                        "Organism|Lumen|*|Aciclovir*"
+                    ],
+                    "Exclude":[
+                        "Organism|Lumen|Feces|Aciclovir*"
+                    ]
+                },
+                {
+                    "Name": "Excreted",
+                    "Include": [
+                        "Organism|Kidney|Urine|Aciclovir*",
+                        "Organism|Lumen|Feces|Aciclovir*"
+                    ]
+                },
+                {
+                    "Name": "Rest of Aciclovir",
+                    "Include": [
+                        "Organism|**|Aciclovir*"
+                    ]
+                }
+            ]
+        },
+        {
+            "Name": "Aciclovir Mass Balance focus on Fat",
+            "Molecules": [
+                "Aciclovir"
+            ],
+            "Groupings": [
+                {
+                    "Name": "Fat",
+                    "Include": [
+                        "Organism|Fat|*|Aciclovir",
+                        "Organism|Fat|*|Aciclovir"
+                    ]
+                },
+                {
+                    "Name": "Plasma & Blood Cells",
+                    "Include": [
+                        "Organism|**|Plasma|Aciclovir",
+                        "Organism|**|BloodCells|Aciclovir"
+                    ],
+                    "Exclude":[
+                        "Organism|Lumen|*|Aciclovir"
```

Comment:
Again, Lumen-Exclusion does not have any effect here.
Instead, we should exclude Fat (otherwise it will be represented twice)

Reply:
If Fat was included in a previous field, it won't be included here.
In issue #1118, we suggested the optional field `ExcludePreviousGroupings` which is true by default.

I'll remove all the redundant Lumen exclusions